hs/src/Reach/Connector/ALGO.hs

```diff
@@ -3138,13 +3136,9 @@ cmeth sigi = \case
   CApi who sig _ _ _ _ _ wls -> do
     block_' (bunpack who) $ do
       comment $ LT.pack $ "API: " <> sig
-      comment $ LT.pack $ " ui: " <> show sigi
-      forM_ wls $ \ (w, l) -> do
-        gvLoad GV_currentStep
-        cint $ fromIntegral w
-        op "=="
-        code "bnz" [l]
-      op "err"
+      comment $ LT.pack $ sigDump sigi
+      let go _ _ = gvLoad GV_currentStep
+      cblt "api" go $ bltM $ M.fromList wls
```

Comment:
This isn't correct. `cblt` assumes the target is on the stack and it will call the function for the entry. I think you want
```
gvLoad GV_currentStep
let go _ l = code "b" [l]
cblt "api" go $ bltM $ M.fromList wls
```

Reply:
If the code you wrote didn't crash CI, then you need better tests

Reply:
wrote a better test that uses remote calls